code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -123,4 +125,13 @@
 
 /datum/deepfryer_special/carrotfries
 	input = /obj/item/reagent_containers/food/snacks/grown/carrot/wedges
-	output = /obj/item/reagent_containers/food/snacks/carrotfries
\ No newline at end of file
+	output = /obj/item/reagent_containers/food/snacks/carrotfries
+
+/datum/deepfryer_special/fried_vox
+	input = /obj/item/organ/external
+	output = /obj/item/reagent_containers/food/snacks/fried_vox
+
+/datum/deepfryer_special/fried_vox/validate(var/obj/item/I)
+	..()
```

Comment:
Oh meant that you can replace the stuff in the if.
Now it will execute the parents version. But the return value is not used in this case.
```
if(!..())
    return FALSE
```
is what I meant